azerothcore/azerothcore-wotlk
# fix(DB/Creature): Ferra Boss now has its correct loot

## Changes Proposed:
-  add missing loot to npc 14308 (Ferra)

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/15780

## SOURCE:

https://www.wowhead.com/wotlk/npc=14308/ferra
## Tests Performed:

- tested in game


## How to Test the Changes:


1. open the database
2. I added the missing loot to the boss
3. add npc 14308 and see

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
